app/src/storages/Realm.ts

```diff
@@ -3,25 +3,26 @@ import Realm from 'realm';
 const TestSchema = {
   name: 'Test',
   properties: {
-    _id: 'int',
-    value: 'string',
+    dict: '{}',
   },
-  primaryKey: '_id',
+};
+
+type Test = {
+  dict: Realm.Dictionary;
 };
 
 const realm = new Realm({
   schema: [TestSchema],
+  deleteRealmIfMigrationNeeded: true,
 });
 
-realm.write(() => {
+const object = realm.write(() => {
   realm.deleteAll();
-  realm.create('Test', {
-    _id: 1,
-    value: 'hello',
-  });
+  return realm.create<Test>('Test', {dict: {k: 'hello'}});
 });
 
+const dict = object.dict;
+
 export function getFromRealm() {
-  const object = realm.objectForPrimaryKey('Test', 1);
-  return object.value;
+  return dict.k;
```

Comment:
This is syntactic sugar for getting the value associated with the `"k"` key in the dictionary, it's not already read from the database, nor cached, as the `Realm.Dictionary` lazy-loading values from the database and getting this "k" hits a Proxy "get" trap:
- On `realm@11`: https://github.com/realm/realm-js/blob/v11.0.0-rc.0/lib/dictionary.js#L22-L48
- On `realm@12`: https://github.com/realm/realm-js/blob/main/packages/realm/src/Dictionary.ts#L46-L55